packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -120,9 +120,10 @@ export class Accordion extends GOVUKFrontendComponent {
     super()
 
     if (!($module instanceof HTMLElement)) {
-      throw new ElementError('$module', {
+      throw new ElementError({
         componentName: 'Accordion',
-        element: $module
+        element: $module,
+        identifier: 'Root element (`$module`)'
```

Comment:
Can't remember if we said to keep `$module` versus @romaricpascal's HTML preference?

I'm happy with `$module` though as only `initAll()` is guaranteed to use this:

```suggestion
        identifier: 'Root element (`<div data-module="govuk-accordion">`)'
```

Reply:
On this one, I was going by the comment here: https://github.com/alphagov/govuk-frontend/issues/4299#issuecomment-1753245288, which points out the same sort of issue with using the HTML.

Reply:
Great minds, thanks for confirming